src/server/worldserver/worldserver.conf.dist

```diff
@@ -1835,6 +1842,20 @@ Visibility.Notify.Period.InInstances  = 1000
 Visibility.Notify.Period.InBGArenas   = 1000
 
 #
+#    GuarderSight
+#        Description: Distance to see guard creatures
+#        Default:     50.0
+
+GuarderSight = 50.0
```

Comment:
Well, I added it because of the changes in https://github.com/TrinityCore/TrinityCore/commit/6487e2f which prompts us to add this to the `worldserver.conf` file, like this:
```
Missing name GuarderSight in config file worldserver.conf, add "GuarderSight = 50.000000" to this file
```